src/main/java/org/folio/rest/util/RestConstants.java

```diff
@@ -27,6 +27,12 @@
       "ebsco", "orderedthroughebsco"
     );
 
+  public static final Map<String, String> FILTER_CUSTOM_MAPPING =
+    Map.of(
+      "true", "custom",
+      "false", "notcustom"
+    );
+
```

Comment:
You will never map to "custom" and "notcustom" values. You may create a list with possible values instead map